docs/apicast-crd-reference.md

```diff
@@ -0,0 +1,210 @@
+# Apicast Custom Resource Reference
+
+## Simple CR Example
+
+```yaml
+apiVersion: saas.3scale.net/v1alpha1
+kind: Apicast
+metadata:
+  name: simple-example
+spec:
+  staging:
+    externalDnsHostname: "apicast-staging.test.stg-saas.3sca.net"
+    image:
+      tag: apicast-v3.8.0-r3
+    replicas: 1
+    env:
+      configurationCache: "30"
+      threescalePortalEndpoint: "http://mapping-service/config"
+    marin3r:
+      enabled: true
+      annotations:
+        marin3r.3scale.net/node-id: apicast-staging
+        marin3r.3scale.net/ports: gateway-http:38080,gateway-https:38443,envoy-metrics:9901
+  production:
+    externalDnsHostname: "apicast-production.test.stg-saas.3sca.net"
+    image:
+      tag: apicast-v3.8.0-r3
+    replicas: 1
+    env:
+      configurationCache: "300"
```

Comment:
Being a field representing time in seconds, shouldnt it be an integer, instead of a string? (applies to examples, docs, CRD)

Reply:
Validations added in 1d9096289bf609153e378ba6ae025774854cbc4f